SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -356,31 +356,26 @@ TSharedPtr<FUnrealType> CreateUnrealTypeInfo(UStruct* Type, uint32 ParentChecksu
 
 		TSharedPtr<FUnrealProperty> PropertyNode = nullptr;
 
+		// It's possible to have duplicate parent properties (they are distinguished by ArrayIndex), so we make sure to look at them
```

Comment:
Why this change?

Reply:
Was planning to discuss this somewhere, but may as well put some sort of explanation here.
This one is potentially dangerous, and I am not sure what I have done here will actually work. Previously to figure out if commands corresponded to funrealproperties, we used the CompatibleChecksum. Unfortunately, this is now computed differently for Cmds and Properties in 4.27, so this method will not work. Instead, I noticed they pointed to the same FProperty, but I don't know if that will always be the case, or if that even is a valid way of finding matching pairs. This needs quite a bit more investigation.

Reply:
tricksy.

Reply:
good news, found out I was dumb, and this is not needed

Reply:
Glad you got to the bottom of it!